### PR TITLE
feat: Support Python 3.10

### PR DESCRIPTION
**Pull request by l0b0**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -34,14 +34,6 @@ repos:
         types: [python]
         require_serial: true
 
-      - id: pretty-format-toml
```
Comment (paulfouquet) [inline .pre-commit-config.yaml:37]: Was it duplicate with `prettier`?

Comment (l0b0) [reply .pre-commit-config.yaml:37]: It's explained in the commit message:

> - Disable pretty-format-toml incompatible with latest Poetry lock format

Comment (paulfouquet) [reply .pre-commit-config.yaml:37]: Ah, my bad, I haven't seen the `...` next to the first line of your commit message to show the following lines!

Comment (l0b0) [reply .pre-commit-config.yaml:37]: Yeah, GitHub have done a good job of making commit messages much less useful.

**Pull request merged**